lib/utils/app-installation.ts

```diff
@@ -32,4 +32,29 @@ async function isAppInstalled(
   }
 }
 
-module.exports.isAppInstalled = isAppInstalled
+export async function installApp(
+  client: ClientAPI,
+  {
+    spaceId,
+    environmentId,
+    appId
+  }: {
+    spaceId: string
+    environmentId: string | string[]
+    appId: string
+  }
+): Promise<void> {
+  const environments = Array.isArray(environmentId)
+    ? environmentId
+    : [environmentId]
+
+  for (const environmentId of environments) {
+    await client.rawRequest({
```

Comment:
I am wondering if this function could be used instead of the rawRequest. https://github.com/contentful/contentful-management.js/blob/97adca455dd85a8552ae6510ff560e7bb058b07f/lib/entities/app-installation.ts#L47

Reply:
That function seems to do an upsert and it seems like you need an existing appInstallation in the environment which we don't have. 
Also, as we are not using the plain client we would need to fetch space and env before updating, making the request count a lot higher.